rpc/documentation/clientusage.md

```diff
@@ -60,14 +63,32 @@ import (
 	"github.com/decred/dcrd/dcrutil"
 )
 
-var certificateFile = filepath.Join(dcrutil.AppDataDir("dcrwallet", false), "rpc.cert")
+var (
+	certificateFile      = filepath.Join(dcrutil.AppDataDir("dcrwallet", false), "rpc.cert")
+	walletClientCertFile = filepath.Join(dcrutil.AppDataDir("dcrwallet", false), "client.pem")
+	walletClientKeyFile  = filepath.Join(dcrutil.AppDataDir("dcrwallet", false), "client-key.pem")
+)
 
 func main() {
-	creds, err := credentials.NewClientTLSFromFile(certificateFile, "localhost")
+	serverCAs := x509.NewCertPool()
+	serverCert, err := ioutil.ReadFile(certificateFile)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	if !serverCAs.AppendCertsFromPEM(serverCert) {
+		fmt.Println("AppendCertsFromPEM error")
```

Comment:
I would suggest:

```Go
fmt.Printf("no certificates found in %s\n", certificateFile)
```

That is really what the issue is here and I think is a much more obvious error from a user perspective.

Reply:
Resolved with https://github.com/decred/dcrwallet/pull/1907/commits/5bf0244831151ef4e9a5dd09786d28e9feb21242